Resources/Private/Partials/Detail/Opengraph.html

```diff
@@ -1,46 +1,57 @@
 <html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers"
 	  xmlns:n="http://typo3.org/ns/GeorgRinger/News/ViewHelpers"
 	  data-namespace-typo3-fluid="true">
-
-<f:if condition="{newsItem.alternativeTitle}">
-	<f:then><n:metaTag property="og:title" content="{newsItem.alternativeTitle}" /></f:then>
-	<f:else><n:metaTag property="og:title" content="{newsItem.title}" /></f:else>
+<f:comment><!--@todo: Rename partial to e.g. "Metadata" at next major release because this would be breaking--></f:comment>

```

Comment:
I don't know exactly what Georg's preference is here, but for my own opinion: please avoid TODOs in the code. If there should be a rename of this template file, please create a pull request after this is merged (and make a note that merging that pull request constitutes a breaking change).